.gitignore

```diff
@@ -0,0 +1,39 @@
+*~
+*.pyc
+*.pyo
+*.tmp*
+*.mo
+*.egg
+*.EGG
+*.egg-info
+*.EGG-INFO
+*.kpf
+*.swp
+*.wpr
+.*.cfg
+.installed.cfg
+.mr.developer.cfg
+.eggs/
+.hg/
+.bzr/
+.svn/
+.project
+.pydevproject
```

Comment:
I suggest not including project specific ignores like IDEs or OSs as well as VCS not in use

Reply:
@davilima6 done with commit abbe3cf